### ISSUE TITLE
picochess.init

### ISSUE THREAD

**Issue by Laterunner** [jromang/picochess#200]
Hi,  to get some information about the DGT Board and BT/serial connection state, I recommend to insert the following lines in picochess.init before the startup call (line 31)
# ----------- # this is inserted: ------------------------

```
### for testing
# hcitool scan
#  rfcomm -a
# hciconfig
# hcitool scan

 var_ip=`hostname -I|grep " "|cut -d " " -f1`
 var_hci=`/usr/sbin/hciconfig|grep "^hci"|cut -d":" -f1`
 var_mac=`rfcomm -a|grep "channel"|cut -d" " -f2`
 var_port=`rfcomm -a|grep "channel"|cut -d" " -f1`
 var_connection=`rfcomm -a|grep "rfc"|cut -d" " -f5`


 echo ""
 echo "IP:                                      "$var_ip
 echo "hcix:                                    "$var_hci
 echo "MAC Adresse eBoard:                       "$var_mac
 echo "ACM0/ttyUSB/rfcomm0:                     "$dgt_port
 echo "BT Port Status (clean/connected/closed): "$var_connection
 echo "NOW PLEASE WAIT 15 secs"
 echo ""
 sleep 15
 # ------------end od insertion--------------------------
```

I inserted these lines  on my odroid system because I had a bluetooth connection problem with earlier versions of picochess (up to 69).  This issue only occurred after a fresh boot and with the first call of picochess. I discussed this in an earlier, now closed thread (thread # 145).

Kind regards 
Laterunner

**Comment by LocutusOfPenguin** [comment]
see my comments on #198 
On our images, this folder isnt used anymore.

**Comment by Laterunner** [comment]
Yes ok, but the file was useful for manually installed systems
Kind regards 
Laterunner

**Comment by LocutusOfPenguin** [comment]
closed on v0.78
